<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Travel Simulation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="8,2">
    <item>
     <widget class="QTextBrowser" name="logBrowser"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="depatureLabel">
          <property name="text">
           <string>出发地</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="departureBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="destinationLabel">
          <property name="text">
           <string>目的地</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="destinationBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="passLable">
        <property name="text">
         <string>途径地列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="passList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QComboBox" name="cityBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="addCity">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteCity">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
        <item>
         <widget class="QLabel" name="startTimeLabel">
          <property name="text">
           <string>出发日期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="startTime"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7">
        <item>
         <widget class="QLabel" name="expectedTimeLabel">
          <property name="text">
           <string>预期时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="expectedTime"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,7">
        <item>
         <widget class="QLabel" name="endTimeLabel">
          <property name="text">
           <string>到达时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="endTime"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
        <item>
         <widget class="QLabel" name="budgetLabel">
          <property name="text">
           <string>预算金额</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="budgetEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>规划策略</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="strategyBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="planButton">
          <property name="text">
           <string>开始规划</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="simButton">
          <property name="text">
           <string>开始模拟</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
